docs/src/lib/sets/SparsePolynomialZonotope.md

```diff
@@ -18,6 +18,6 @@ ngens_dep(::SparsePolynomialZonotope)
 ngens_indep(::SparsePolynomialZonotope)
 nparams(::SparsePolynomialZonotope)
 order(::SparsePolynomialZonotope)
-linear_map(::AbstractMatrix, ::SparsePolynomialZonotope)
+linear_map(::Union{Real, AbstractMatrix, AbstractVector, LinearAlgebra.UniformScaling}, ::SparsePolynomialZonotope)
```

Comment:
Do you know if the methods also work for `AbstractVector` and `UniformScaling`?

Reply:
for `UniformScaling` yes. For `AbstractVector`  tbh I don't even understand why it's there (I copied that signature from the definition for zonotope), I cannot think of a case where you would pass a vector (especially since a vector is not a linear map)

Reply:
Then remove it :)

Reply:
only from here or also from elsewhere in the package?

Reply:
in this PR I'll remove it from here only, we can open an issue about this and address in separate PR

Reply:
Yes, only from here.